api/views.py

```diff
@@ -82,8 +82,8 @@ class PumpingViewSet(viewsets.ModelViewSet):
     queryset = models.Pumping.objects.all()
     serializer_class = serializers.PumpingSerializer
     filterset_class = filters.PumpingFilter
-    ordering_fields = ("amount", "time")
-    ordering = "-time"
+    ordering_fields = ("amount", "duration", "end", "start")
+    ordering = "-start"
```

Comment:
For now lets follow the same default ordering as `Feeding` and set this to `-end`.

Reply:
fixed with [b8a1e39987cfcad5a7d59cb60b0e153812dc2b94](https://github.com/babybuddy/babybuddy/pull/665/commits/b8a1e39987cfcad5a7d59cb60b0e153812dc2b94)